README.md

```diff
@@ -38,6 +38,7 @@ This will walk through the simplest path of spinning up a production test enviro
 ```
 git clone https://github.com/theforeman/forklift.git
 cd forklift
+ansible-galaxy collection install -r requirements.yml
```

Comment:
Does this work with Ansible 2.7 or should the Requirements section be updated?

Reply:
It's 2.9+, yeah